City-of-Pori/pori-kada
# PORI-369 Fixed RSS-displaying all around the site, changed Visitpori …

…front page to use section instead of a page

**ATTENTION!** In order for everything to work, this requires removing "visitpori -> <front>" url redirect from production and change in the site configuration of visitpori.fi so that the front page of the site is visitpori instead of visit-pori. Also you need to edit the visitpori section node and check under "Ajankohtaisvirta ja tapahtumawidgetit" the checkboxes for "Uutiset", "Facebook", "Instagram" and "Youtube". Also the visitpori-section node is marked to wrong domain so that needs to be changed.

To test:

1. drush fra -y && drush cc all
2. Go to remove url redirect https://local.visitpori.fi/admin/config/search/redirect/list/visitpori that says visitpori -> <front>
3. Go to edit visitpori front page https://local.visitpori.fi/admin/config/system/site-information and change it from visit-pori to visitpori
4. Go to edit the visitpori section node https://local.visitpori.fi/node/3183/edit and under "Ajankohtaisvirta ja tapahtumawidgetit" check "Uutiset", "Facebook", "Instagram" and "Youtube".
5. Go to https://local.pori.fi/ and make sure there is rss-icon under the news block and it links to https://local.pori.fi/news/rss.xml
6. Go to https://local.pori.fi/asuminen-ja-ymparisto and make sure there is rss-icon under the news block and it links to https://local.pori.fi/term/61/rss.xml
7. Go to https://local.visitpori.fi/ and make sure there is rss-icon under the news block and it links to https://local.visitpori.fi/section/3183/rss.xml
8. Go to https://local.pori.fi/asuminen-ja-ymparisto/asuminen and edit the node. Under "Ajankohtaisvirta ja tapahtumawidgetit" uncheck the box "Teeman pääsivu". Save the node.
9. Make sure the page https://local.pori.fi/asuminen-ja-ymparisto/asuminen doesn't have tags or the rss-icon visible, only title and blocks that link to different pages.
10. Go to https://local.pori.fi/nuoret and make sure there is rss-icon under the news block and it links to https://local.pori.fi/term/57/rss.xml
